packages/xstate-store/src/types.ts

```diff
@@ -298,7 +298,7 @@ export interface InspectedActorEvent extends BaseInspectionEventProperties {
 export type ActorRefLike = {
   sessionId: string;
   // https://github.com/statelyai/xstate/pull/5037/files#r1717036732
-  send: (...args: never) => void;
+  send: (event: any) => void;
```

Comment:
@Andarist The `...args: never` part doesn't work with this:

```ts
const { inspect } = createBrowserInspector();

// ...

store.inspect(inspect); // fails on actorRef.send(…)
```

Reply:
You could also fix it like this:
<details>
<summary>Alternative patch</summary>

```diff
diff --git a/packages/core/src/inspection.ts b/packages/core/src/inspection.ts
index ec61c3aa1b..1e3114d4d5 100644
--- a/packages/core/src/inspection.ts
+++ b/packages/core/src/inspection.ts
@@ -35,7 +35,7 @@ interface InspectedMicrostepEvent extends BaseInspectionEventProperties {
   type: '@xstate.microstep';
   event: AnyEventObject; // { type: string, ... }
   snapshot: Snapshot<unknown>;
-  _transitions: AnyTransitionDefinition[];
+  _transitions: unknown[];
 }
 
 export interface InspectedActionEvent extends BaseInspectionEventProperties {
@@ -51,7 +51,7 @@ export interface InspectedEventEvent extends BaseInspectionEventProperties {
   // The source might not exist, e.g. when:
   // - root init events
   // - events sent from external (non-actor) sources
-  sourceRef: AnyActorRef | undefined;
+  sourceRef: ActorRefLike | undefined;
   event: AnyEventObject; // { type: string, ... }
 }
 
diff --git a/packages/core/src/types.ts b/packages/core/src/types.ts
index c84c91281d..463897aff2 100644
--- a/packages/core/src/types.ts
+++ b/packages/core/src/types.ts
@@ -1981,10 +1981,9 @@ export interface ActorRef<
 
 export type AnyActorRef = ActorRef<any, any, any>;
 
-export type ActorRefLike = Pick<
-  AnyActorRef,
-  'sessionId' | 'send' | 'getSnapshot'
->;
+export type ActorRefLike = Pick<AnyActorRef, 'sessionId' | 'getSnapshot'> & {
+  send: (...args: never) => void;
+};
 
 export type UnknownActorRef = ActorRef<Snapshot<unknown>, EventObject>;
 
diff --git a/packages/xstate-store/src/types.ts b/packages/xstate-store/src/types.ts
index 0ee864a93c..4aaed3491e 100644
--- a/packages/xstate-store/src/types.ts
+++ b/packages/xstate-store/src/types.ts
@@ -298,7 +298,7 @@ export interface InspectedActorEvent extends BaseInspectionEventProperties {
 export type ActorRefLike = {
   sessionId: string;
   // https://github.com/statelyai/xstate/pull/5037/files#r1717036732
-  send: (event: any) => void;
+  send: (...args: never) => void;
   getSnapshot: () => any;
 };
```

</details>

I don't mind either solution... although I really like how the `...args: never` approach discovered here further inconsistencies in types. Something that `any` didn't 😉

Reply:
btw. let's add a type test for this combination of things